### PR TITLE
New release v2.8.0 - No more Python 3.7

### PR DESCRIPTION
**Pull request by i-oden**
## Read this before submitting the PR

1. Always create a Draft PR first
2. Go through sections 1-5 below, fill them in and check all the boxes
3. Make sure that the branch is updated; if there's an "Update branch" button at the bottom of the PR, rebase or update branch.
4. When all boxes are checked, information is filled in, and the branch is updated: mark as Ready For Review and tag reviewers (top right)
5. Once there is a submitted review, implement the suggestions (if reasonable, otherwise discuss) and request an new review.

If there is a field which you are unsure about, enter the edit mode of this description or go to the [PR template](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/.github/pull_request_template.md); There are invisible comments providing descriptions which may be of help.

## 1. Description / Summary

New release

## 2. Jira task / GitHub issue

[_Link to the github issue or add the Jira task ID here._](https://scilifelab.atlassian.net/jira/software/projects/DDS/boards/13?selectedIssue=DDS-2130)

## 3. Type of change

What _type of change(s)_ does the PR contain?

**Check the relevant boxes below. For an explanation of the different sections, enter edit mode of this PR description template.**

- [x] New feature
  - [ ] Breaking: _Why / How? Add info here._ 
  - [x] Non-breaking 
- [ ] Bug fix 
- [x] Security Alert fix 
  - [x] Package update 
    - [x] Major version update 
- [x] Documentation 
- [ ] Workflow 
- [ ] Tests **only** 

## 4. Additional information

- [x] [Sprintlog](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/SPRINTLOG.md) 
- [ ] Blocking PRs 
  - [ ] Merged 
- [x] PR to `master` branch: _If checked, read [the release instructions](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/docs/procedures/new_release.md)_ 
  - [x] I have followed steps 1-8. 

## 5. Actions / Scans

_Check the boxes when the specified checks have passed._

**For information on what the different checks do and how to fix it if they're failing, enter edit mode of this description or go to the [PR template](https://github.com/ScilifelabDataCentre/dds_cli/blob/dev/.github/pull_request_template.md).**

- [x] **Black**

- [x] **Pylint**

- [x] **Prettier**

- [x] **Yamllint**

- [x] **Tests**

- [x] **TestPyPI**

- [x] **CodeQL**

- [x] **Trivy**

- [x] **Snyk**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+    - `zstandard` from `0.19.0` to `0.22.0`
+- Dependencies (tests): 
+    - `flask` from `2.0.2` to `2.2.5`
+    - `pyfakefs` from `4.5.5` to `5.3.0`
+- Documentation:
+    - Improve documentation related to the testing instance.
+    - Clarify documentation about password management.
+    - Update documentation for python 3.12
+- Fixes:
+    - Remove pinned version of pynstaller in requeriments-dev to solve cve.
```
Comment (i-oden) [inline CHANGELOG.rst:27]:
```suggestion
    - Remove pinned version of pyinstaller in requeriments-dev to solve cve.
```

Comment (i-oden) [reply CHANGELOG.rst:27]: Also I think you can move this to the "dependencies (tests)" part.

Comment (i-oden) [reply CHANGELOG.rst:27]: @rv0lt I can do this change now quickly and you can approve

**Review by rv0lt — APPROVED** [review]

**Comment by rv0lt** [comment]
@aishling-scilifelab Still needs approval from someone else somehow, maybe because I assigned myself to it?

**Comment by i-oden** [comment]
> @aishling-scilifelab Still needs approval from someone else somehow, maybe because I assigned myself to it?

For me it says it's ok to merge (Also Aishling is on holiday). 
Since it's all been previously approved in PRs, I will merge now.

**Comment by rv0lt** [comment]
> > @aishling-scilifelab Still needs approval from someone else somehow, maybe because I assigned myself to it?
> 
> For me it says it's ok to merge (Also Aishling is on holiday). Since it's all been previously approved in PRs, I will merge now.

Uh sorry, I meant to ping you

**Pull request merged**